test/unit/refresh.js

```diff
@@ -169,8 +139,9 @@ describe('Unit tests for swiftserver:refresh', function () {
                               specObj: {
                                 appType: 'scaffold',
                                 appName: 'myapp',
-                                services: {
-                                  objectstorage: [{}]
+                                bluemix: {
+                                  backendPlatform: 'SWIFT',
+                                  objectStorage: [{ serviceInfo: {} }]
```

Comment:
doesn't actually test the key `serviceInfo` exists unless we put something real in the value (not the empty object)

Reply:
not sure if this checks whether `objectStorage` exists or not...